src/common/module.c

```diff
@@ -40,7 +40,12 @@ static int run_command(const char *command)
 
 	if (status < 0) {
 		char error_buf[80];
-		strerror_r(errno, error_buf, sizeof(error_buf));
+		#if defined(__FreeBSD__)
```

Comment:
@xiexingguo if you want to differentiate the XSI and GNU `strerror_r()`,  better off using `STRERROR_R_CHAR_P` instead of `__FreeBSD__`. again, i think `(void)` is a simpler alternative.